book/src/nostr-sdk/03-quickstart.md

```diff
@@ -0,0 +1,34 @@
+## Quickstart
+
+<custom-tabs category="lang">
+
+<div slot="title">Rust</div>
+<section>
+
+Create a client and connect to some relays.
+
+```rust,ignore
+{{#include ../../snippets/nostr-sdk/rust/src/quickstart.rs:5:26}}
```

Comment:
Instead of using the code lines, can you the `ANCHOR`?

https://rust-lang.github.io/mdBook/format/mdbook.html#including-portions-of-a-file